### PR TITLE
Add metas column to Message

### LINKED ISSUE #19: Add metas column to Message
Replace `Metas` by adding `metas` column to `Message` model.

metas column schema: `json`

CheckList:
- [x] add `metas` column to message
- [x] delete `Metas` model, migration, trait and relationship
- [x] remove `metas` config refs from chatsystem's `config`
- [x] update `MessageController@store` token creation by using `message.metas` column to store token
- [x] remove metas refs from  `MessageController@index`

### PR DESCRIPTION
**Pull request by hmreumann**
resolves #19 
A little bit advance for me, but I hope it works. Thanks in advance.

### CODE REVIEW

File: src/Http/Controllers/MessageController.php [hunk 1]
```diff
@@ -146,16 +141,14 @@ public function store(Request $request)
 
       if ($message->wasRecentlyCreated) {
         if ($token) {
-          $meta = ['name' => 'token', 'value' => $token];
-          $message->addMeta($meta, $meta);
+          $message->update(['metas->token' => $token]);
```
Comment (myckhel) [inline src/Http/Controllers/MessageController.php:144]: we can do better than this.

we can create metas->token when message model is being created.
running an update will result to an unnecessary update query.

we could do all at once during creation.

**Review by myckhel — CHANGES_REQUESTED** [review]
Finally test the api endpoint with postman and fix error when creating message.

If you cannot test with postman.

i will just fix the error.

i fixed the error and reverted so you can see the error.

Apart from that every thing worked fine when tested.

**Comment by hmreumann** [comment]
Hallo, I messed with the commits, I didn't know you could change things in my forked repository, but great. So i did your changes again. You will see the same commits repeted.
The last commit is what I added to the firstorcreate message function.
I tried to do a pull request again but it seems its not necessary.
Best Regards, Hernán.

**Comment by myckhel** [comment]
Great job boss!

**Review by myckhel — APPROVED** [review]

**Comment by hmreumann** [comment]
Excelent! Obviously a better sintax. Haha!!

**Pull request merged**